apps/api/src/scraper/WebScraper/single_url.ts

```diff
@@ -85,18 +89,18 @@ function getScrapingFallbackOrder(
   });
 
   let defaultOrder = [
-    !process.env.USE_DB_AUTHENTICATION ? undefined : "fire-engine",
-    !process.env.USE_DB_AUTHENTICATION ? undefined : "fire-engine;chrome-cdp",
-    "scrapingBee",
-    process.env.USE_DB_AUTHENTICATION ? undefined : "playwright",
-    "scrapingBeeLoad",
+    useFireEngine ? "fire-engine" : undefined,
+    useFireEngine ? "fire-engine;chrome-cdp" : undefined,
+    useScrapingBee ? "scrapingBee" : undefined,
+    useScrapingBee ? "scrapingBeeLoad" : undefined,
+    useDatabaseAuth ? undefined : "playwright",
     "fetch",
   ].filter(Boolean);
 
   if (isWaitPresent || isScreenshotPresent || isHeadersPresent) {
     defaultOrder = [
       "fire-engine",
-      process.env.USE_DB_AUTHENTICATION ? undefined : "playwright",
+      useDatabaseAuth ? undefined : "playwright",
```

Comment:
I do find this a little confusing. Why would we not use Playwright when there's database auth enabled?

Reply:
just changed to useFireEngine instead. No need for checking the DB here.